e2e/list-view-grid.e2e-spec.ts

```diff
@@ -40,8 +41,6 @@ describe('List view grid', () => {
   });
 
   it('should match the previous screenshot with column chooser open (screen: xs)', (done) => {
```

Comment:
You removed the portion here that sets this test to the `xs` size.

Reply:
good catch! fixed.